### DISCUSSION TITLE
Unknown Problem when Trying to Create my Stripe Connect Account

### DISCUSSION THREAD

**Discussion by psygo** [community/community#32314] [Other Feature Feedback, Questions, & Ideas] [answered]
> The problem I describe below also happened when I first clicked on the `Create a Stripe Connect account` link.

This is what I get after going into `Stripe Connect account`'s link:

![image](https://user-images.githubusercontent.com/11823725/189145262-5a2629c9-b500-45c1-ad2e-b4fa1bedd65f.png)

Then, if I try to go into my Stripe Dashboard, I'm met with something that looks like a sign-in page that keeps being loaded:

![Screenshot_2022-09-08-11-14-58_5120x1080 - Censored](https://user-images.githubusercontent.com/11823725/189146426-5aac555f-07cf-4f51-b8f5-a6fb637603ef.png)

After a while, the loading stops, and I get this error, which is the same I got from when trying to create a Stripe Connect account:

![image](https://user-images.githubusercontent.com/11823725/189146058-fc47fc24-64da-4e23-a5d5-292f8b10175b.png)

**Comment by diamant3** [comment, marked as answer]
update your browser or use other browser, please send a ticket [here](https://support.github.com) if didn't fix

**Reply by psygo** [reply]
Nice, now I'm able to submit my application!

**Reply by diamant3** [reply]
That's good to hear. Good luck with your approval! :relaxed:
